### DISCUSSION TITLE
[FF7 STEAM Spanish] Game Crashes on ingame menu

### DISCUSSION THREAD

**Discussion by Damyos** [julianxhokaxhiu/FFNx#735] [General]
Just installed the game (spanish version), installed FFNX, and the game crashes when you enter the menu and go to edit controll options. Just check with and without FFNX and crashes only with FFNX installed, checked with FF7 English version and FFNX installed and the crash didnt happen.

**Comment by julianxhokaxhiu** [comment]
Thanks for reporting it, might be probably related to the latest implementations we did on FF7. I'll try to have a look and let you know.

**Reply by julianxhokaxhiu** [reply]
I did test it here and it seems to work fine
![image](https://github.com/user-attachments/assets/3698745c-41b5-452f-8919-65c6c8dad750)

Can you please post your FFNx.log file?

**Comment by Damyos** [comment]
just check again, with a clean new install and the game keeps crashing, here is my log file
[FFNx.log](https://github.com/user-attachments/files/17173350/FFNx.log)

**Reply by julianxhokaxhiu** [reply]
Thank you very much. I was able to reproduce and I already pushed a fix. Please try the next canary as soon as it is available.

**Reply by Damyos** [reply]
the crash is fixed with the last canary build. Thank you!

**Reply by julianxhokaxhiu** [reply]
Glad to hear that. Thanks for coming back and confirming :) Enjoy!
